index.bs

```diff
@@ -2932,6 +2969,8 @@ an assertion. Its {{PublicKeyCredentialRequestOptions/challenge}} member MUST be
         USVString                            rpId;
         sequence<PublicKeyCredentialDescriptor> allowCredentials = [];
         DOMString                            userVerification = "preferred";
+        DOMString                            attestation = "none";
+        sequence<DOMString>                  attestationFormats = [];
```

Comment:
It is really not clear what these parameters are used for during assertion. Does this mean the authenticator can re-attest its primary credential, or is this solely to influence the client w.r.t. the DPK extension?

Reply:
> It is really not clear what these parameters are used for during assertion. Does this mean the authenticator can re-attest its primary credential, or is this solely to influence the client w.r.t. the DPK extension?

I think it's just for the primary credential right, because the extension has it's own input parameters for attestation and attestationFormats....